introduction.md

```diff
@@ -1,3 +1,12 @@
 Hello! My name is Mike Robards, and I am Mastering GitHub with Code School!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?